dev/scripts/start_with_docker.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+##Set Your proxy and cache path here
+HTTP_PROXY='http://10.24.221.169:911'
+HTTPS_PROXY='http://10.24.221.169:911'
```

Comment:
suggest to not set this proxy by default for user, but we can use this env in our own CI.